_posts/2017-08-21-Write-Efficient-Deep-Learning-Operators-with-TVM-A-Depthwise-Convolution-Example.md

```diff
@@ -0,0 +1,412 @@
+---
+layout: post
+title:  "Write Efficient Deep Learning Operators with TVM: A Depthwise Convolution Example"
+date:   2017-08-21
+author: Yuwei Hu
+---
+
+This blog aims to share some guidelines and tricks of using TVM to write efficient deep learning operators.
+It is a summary of what I have learned from the experience of writing the depthwise convolution operator, i.e.
+[topi.nn.depthwise_conv2d_nchw](http://docs.tvmlang.org/api/python/topi.html#topi.nn.depthwise_conv2d_nchw).
+
+For the design philosophy of depthwise convolution, you may refer to [the Xception paper](https://arxiv.org/abs/1610.02357).
```

Comment:
Depthwise convolution is an important building blocks of modern architectures, it is used in architectures such as Xception[1](add citation in the end) and MobileNet[2]. The computation pattern of depthwise conv can be described by (add tvm declaration snippets)